### ISSUE TITLE
Presentation mode button is missing

### ISSUE THREAD

**Issue by Aarass** [shivaprsd/doqment#23]
I can't find a way to enter presentation mode. Is it removed in the latest version? Why?

**Comment by shivaprsd** [comment]
Did you check the 》menu on the top right corner? Presentation mode has been under that menu since a long time.

If it is not even there, can you give more details (browser & OS version, *doqment* version) and a screenshot? Thanks!

**Comment by Aarass** [comment]
Yes, it was there since like two weeks ago.
I'm using Chrome on Windows 11
doqment version 0.9.1

<img width="188" alt="Screenshot 2024-12-02 160715" src="https://github.com/user-attachments/assets/f07b93b6-a1a1-41d6-b59d-7dfcd67f3eb5">

**Comment by shivaprsd** [comment]
Indeed, you are right! Since the last Chrome update, the viewer is loaded in an `iframe`, and I didn't explicitly set its permission to request fullscreen. The PDF.js library was reading the (lack of) permission and hiding the button.

It is a straight-forward fix, I will update and push a new minor release this week. Thanks for bringing this to my notice!

**Comment by shivaprsd** [comment]
@Aarass Meanwhile, just out of curiosity: do you use presentation mode to actually present PDFs on a screen/projector, or only for fullscreen reading?

Because with `Page Scrolling` mode, going fullscreen with `F11`/Chrome menu and hiding the PDF toolbar with `F3` achieves _nearly_ the same effect (that's three steps instead of one, but still).

**Comment by Aarass** [comment]
I use it only for fullscreen reading.

The three steps you mentioned almost work. However, the issue I’m having is that I can see the scrollbar at almost all zoom levels. The automatic zoom options leave a small amount of scrolling for some reason (I suspect it’s due to the padding around the page). The scrollbar disappears when I choose a small zoom level, like 50%, but then, what’s the point?

**Comment by shivaprsd** [comment]
I agree, it is not the exact same experience. As I said, the fix is on the way. Thanks for your patience!

(By the way, I think the zoom level is more like `Page Fit` in Presentation mode. And you cannot change it, so the above trick is useful if someone also needs to zoom in fullscreen, if they don't mind the scrollbars)

**Comment by shivaprsd** [comment]
This is now fixed in [v0.9.2](https://github.com/shivaprsd/doqment/releases/tag/v0.9.2). The update should be live on the store in a few hours. Sorry for the delay!
